pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -2102,8 +2108,8 @@ def allocate_budget_to_maximize_response(
             The total budget to be allocated.
         time_granularity : str
             The granularity of the time periods (e.g., 'daily', 'weekly', 'monthly').
-        num_days : int
-            The number of days over which the budget is to be allocated.
+        periods : float
```

Comment:
Same comment as above. Why `periods` instead of `num_periods`?

Reply:
Made the changes!